package.json

```diff
@@ -27,12 +27,11 @@
     "start:with-stub-api": "./stub-api/start-with-stub-api.sh",
     "start:production": "NODE_ENV=production ALLOW_INSECURE_SESSION=true npm run -s build && node dist/main.js",
     "push": "NODE_ENV=production npm run -s build && cf push",
-    "clean": "find dist -maxdepth 1 ! -name downloads ! -name dist -exec rm -r {} \\; || echo 'already clean'"
+    "clean": "find dist -maxdepth 1 ! -name downloads ! -name dist -exec rm -r {} \\; || echo 'already clean'; echo '{}' > dist/manifest.json"
```

Comment:
I don't think leaving this shell hack here is best, before merging I will refactor into some pure JS in the webpack config.
https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback